tcmur_cmd_handler.c

```diff
@@ -2308,6 +2318,312 @@ static int handle_inquiry(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	return ret;
```

Comment:
In this file below, if the function you are adding is only used in this file make the
function static.